pkg/core/provider.go

```diff
@@ -7,6 +6,7 @@ import (
 	"github.com/johnfercher/maroto/v2/pkg/props"
 )
 
+// Provider is the abstraction to create a pdf.
```

Comment:
// Provider is the abstraction of a document creator provider.

Reply:
done